packages/peregrine/lib/List/item.js

```diff
@@ -1,61 +1,43 @@
-import React, { useCallback, useMemo } from 'react';
-import PropTypes from 'prop-types';
+import React, { useMemo } from 'react';
+import { any, bool, func, number, oneOfType, shape, string } from 'prop-types';
 
 import fromRenderProp from '../util/fromRenderProp';
 
-const isString = item => typeof item === 'string';
-
-const getChild = item => (isString(item) ? item : null);
-
 const Item = props => {
     const {
-        uniqueID: key,
         classes,
         hasFocus,
         isSelected,
         item,
         itemIndex,
         render,
-        updateSelection,
-        setFocus,
         ...restProps
     } = props;
-    const onClick = useCallback(() => updateSelection(key), [
```

Comment:
Don't we need this?

Reply:
It is [passed in by the parent](https://github.com/magento/pwa-studio/blob/supernova/900_edit_item_part1/packages/peregrine/lib/List/items.js#L99) (`Items`).